src/interfaces/earnings.ts

```diff
@@ -265,13 +261,11 @@ export class EarningsInterface<C extends ChainId> extends ServiceInterface<C> {
       return dayData;
     });
 
-    const result: AssetHistoricEarnings = {
```

Comment:
Felt this was unnecessary as we can see the return type in the function signature, so no need to assign a value for explicitness IMO